### DISCUSSION TITLE
Backend getting SQLAlchemy missing import error

### DISCUSSION THREAD

**Discussion by cardosop** [fastapi/full-stack-fastapi-template#1147] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
docker compose up
```

### Description

Hi guys, 

Hope you are well.
First time I am here, I am trying to run the application for the first time but I am getting this following error:

backend-1   | Traceback (most recent call last):
backend-1   |   File "/app/app/backend_pre_start.py", line 4, in <module>
backend-1   |     from sqlmodel import Session, select
backend-1   |   File "/usr/local/lib/python3.10/site-packages/sqlmodel/__init__.py", line 117, in <module>
backend-1   |     from .main import Field as Field
backend-1   |   File "/usr/local/lib/python3.10/site-packages/sqlmodel/main.py", line 81, in <module>
backend-1   |     from .sql.sqltypes import GUID, AutoString
backend-1   |   File "/usr/local/lib/python3.10/site-packages/sqlmodel/sql/sqltypes.py", line 5, in <module>
backend-1   |     from sqlalchemy.dialects.postgresql import UUID
backend-1   | ModuleNotFoundError: No module named 'sqlalchemy.dialects.postgresql'

Its important to say that I didnt add any coded to the project and I had followed all procedures listed here:
https://github.com/tiangolo/full-stack-fastapi-template?tab=readme-ov-file#how-to-use-a-private-repository
https://github.com/tiangolo/full-stack-fastapi-template/blob/master/development.md#change-the-development-domain

My code is available at: https://github.com/cardosop/my-full-stack

I really appreacetiate any assistance.

Thanks in advance!!!

### Operating System

Linux

### Operating System Details

ubuntu 22.04

### Python Version

Python 3.12.2

**Comment by YuriiMotov** [comment]
I think this was probably temporary problem in sub-dependencies.
Works well current version
